experiment/compatibility-versions/common.sh

```diff
@@ -235,6 +235,30 @@ build_prev_version_bins() {
   echo "Finished building e2e.test binary from ${PREV_RELEASE_BRANCH}."
 }
 
+download_prev_version_bins() {
+  local version=$1
+  wget https://dl.k8s.io/release/$(curl -Ls https://dl.k8s.io/release/stable-${version}.txt)/kubernetes-test-$(go env GOOS)-$(go env GOARCH).tar.gz
```

Comment:
TODO: this script as-written will only work on a linux host, but it should work on mac too.

For that you need to split binaries being downloaded for kind (GOOS should always be linux) versus for the host (like the test binaries)